tests/framework/cli/test_starters.py

```diff
@@ -1324,3 +1324,92 @@ def test_invalid_names(self, fake_kedro_cli, name):
             "Kedro project names must contain only alphanumeric symbols, spaces, underscores and hyphens and be at least 2 characters long"
             in result.output
         )
+
+
+class TestParseYesNoToBools:
```

Comment:
Could we move the tests for `parse_tools_input` into their own class as well? For consistency